### DISCUSSION TITLE
Inquiry about Live Helper Chat API Capabilities for Temporary Chat Rooms

### DISCUSSION THREAD

**Discussion by RushAlmosa** [LiveHelperChat/livehelperchat#2020] [Q&A]
Dear Live Helper Chat Community,

I am considering integrating Live Helper Chat as a chat engine within my software tool, and I am particularly interested in the capabilities of your REST API. I would like to know if it is possible to use your API to create temporary chat rooms for interactions between a registered user and a guest who is not registered on the platform. This feature is crucial for my application's user experience.

Additionally, I would appreciate any guidance or documentation you could provide about implementing this functionality using your API. I look forward to exploring the potential of incorporating Live Helper Chat into my software.

Thank you for your assistance.

Best regards,
Rush

**Comment by remdex** [comment]
Hi,

Well there is no option to have a chat betwen visitors themself, only between opertor and visitor. Present api you find in https://api.livehelperchat.com/#/
